samples/highcharts/chartchooser/treemap/demo.js

```diff
@@ -0,0 +1,79 @@
+const colors = [
+    '#E59866',
+    '#F8C471',
+    '#F7DC6F',
+    '#82E0AA',
+    '#73C6B6',
+    '#85C1E9',
+    '#BB8FCE',
+    '#F1948A',
+    '#B2BABB'
+];
+
+Highcharts.getJSON(
+    'https://raw.githubusercontent.com/mekhatria/demo_highcharts/master/denmarkExportIn2018.json?callback=?',
+
+    function (dataJson) {
+        colors.forEach((e, i) => {
+            dataJson[i].color = e;
```

Comment:
Instead of setting colors programmatically to each point, you can apply `colors` to the top level of the chart configuration. Then on level 1, set `colorByPoint` to `true`.